slidez-fe-core/src/pages/dashboard/edit-profile/components/FormUpdatePassword.tsx

```diff
@@ -0,0 +1,97 @@
+import { faEye, faEyeSlash } from '@fortawesome/free-regular-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Button from '@material-ui/core/Button'
+import { Field, Form, Formik } from 'formik'
+import React, { useState, useRef, useEffect } from 'react'
+import { selectError } from '../../../../containers/user/store'
+import { useAppSelector } from '../../../../hooks'
+import { FormUpdatePasswordProps } from '../editProfileTypes'
+import { UserField } from '../Field'
+import { updatePasswordFieldsValidation } from '../validations'
+import { UpdatePasswordErrors } from './UpdatePasswordErrors'
+
+const FormUpdatePassword = ({
+    initialValuesPassword,
+    handleUpdatePassword,
+    userId,
+    onCloseEditDialog,
+    viewErrors,
+    setViewErrors,
+}: FormUpdatePasswordProps) => {
+    const updatePasswordError = useAppSelector(selectError)
+
+    const handlePwdSubmit = (values: typeof initialValuesPassword) => {
+        console.log(values)
```

Comment:
Have you impelemented password update? Or what is the purpose of this function?

Reply:
I deleted this, thanks